docs_src/general/ContainerNames.md

```diff
@@ -35,17 +35,19 @@ The following table provides the list of the default EdgeX Docker image names to
     |edgexfoundry/device-grove|edgex-device-grove|edgex-device-grove|device-grove|
     |edgexfoundry/device-coap|edgex-device-coap|edgex-device-coap|device-coap|
 === "Security"
-    |Docker image name|Docker container name |Docker network hostname|Docker Compose service name|
+    |Docker image name|Docker container name |Docker network hostname|Docker Compose service name|Note|
     |---|---|---|---|
     |vault|edgex-vault|edgex-vault|vault|
-    |postgress|edgex-kong-db|edgex-kong-db|kong-db|
-    |kong|edgex-kong|edgex-kong|kong|
+    |nginx|edgex-nginx|edgex-nginx|nginx|
+    |postgress|edgex-kong-db|edgex-kong-db|kong-db|DEPRECATED|
+    |kong|edgex-kong|edgex-kong|kong|DEPRECATED|
+    |edgexfoundry/security-proxy-auth|edgex-proxy-auth|edgex-proxy-auth|authproxy|
     |edgexfoundry/security-proxy-setup|edgex-security-proxy-setup|edgex-security-proxy-setup|proxy-setup|
     |edgexfoundry/security-secretstore-setup|edgex-security-secretstore-setup|edgex-security-secretstore-setup|secretstore-setup|
     |edgexfoundry/security-bootstrapper|edgex-security-bootstrapper|edgex-security-bootstrapper|security-bootstrapper|
 === "Miscellaneous"
-    |Docker image name|Docker container name |Docker network hostname|Docker Compose service name|
+    |Docker image name|Docker container name |Docker network hostname|Docker Compose service name|Note|
     |---|---|---|---|
     |consul|edgex-core-consul|edgex-core-consul|consul|
     |redis|edgex-redis|edgex-redis|database|
-    |edgexfoundry/sys-mgmt-agent|edgex-sys-mgmt-agent|edgex-sys-mgmt-agent|system|
\ No newline at end of file
+    |edgexfoundry/sys-mgmt-agent|edgex-sys-mgmt-agent|edgex-sys-mgmt-agent|system|DEPRECATED|
```

Comment:
This should just be removed. We deprecated it previously
```suggestion
```

Reply:
Removed altogether.